config/default.yml

```diff
@@ -553,6 +553,10 @@ Style/TrivialAccessors:
     - to_s
     - to_sym
 
+Style/UnneededPercentX:
+  # Enforce using %x when the command string spans multiple lines.
+  EnforcePercentXForMultiLine: false
```

Comment:
Hmm, this is getting pretty complex. :-)

Style 2 & 4 doesn't make much sense IMO. Guess we can have just two styles - `backticks`, `percent_x` and `mixed` (backticks on a single-line, %x otherwise). Or we can have just 2 styles and a couple of other variables that affect the behaviour of the `backticks` style - this might make more sense to the users and it will spare us from thinking of odd style names.